proposals/text/7462-rag-evaluation.md

```diff
@@ -0,0 +1,225 @@
+- Title: Proposal for presentation of RAG evaluation results
+- Decision driver: David S. Batista
+- Start Date: 2024-04-03
+- Proposal PR: #7462
+- Github Issue or Discussion: https://github.com/deepset-ai/haystack/issues/7398
+
+# Summary
+
+Add a new component to Haystack allowing users to evaluate the performance of a RAG model.
+
+# Basic example
+
+```python
+
+data  = {
+    "query_id": ["53c3b3e6, 225f87f7, 8ac473ec, 97d284ca"],
+    "reciprocal_rank": [0.378064, 0.534964, 0.216058, 0.778642],
```

Comment:
I wonder where these metric names will come from. One idea is to include the metric name in the output dict of every metric evaluator. I briefly mentioned this idea to @shadeMe  
In a dataframe the metric names are need for naming the columns.